### PR TITLE
Implement some usefull API's

### PR DESCRIPTION
**Pull request by stirante**
I added naming items, setting and getting fuse and explosion radius for creepers, API for Beacon block, setting and getting mob's equipment, setting and getting color of wolf's collar.

### CODE REVIEW

File: src/main/java/org/bukkit/inventory/ItemStack.java [hunk 1]
```diff
@@ -1,408 +1,416 @@
-package org.bukkit.inventory;
```
Comment (md-5) [inline src/main/java/org/bukkit/inventory/ItemStack.java:1]: Seems you broke formatting here.

File: src/main/java/org/bukkit/block/Beacon.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package org.bukkit.block;
+
+import org.bukkit.potion.PotionEffectType;
+
+public interface Beacon extends BlockState{
```
Comment (md-5) [inline src/main/java/org/bukkit/block/Beacon.java]: Need a space before brace here

File: src/main/java/org/bukkit/entity/Creeper.java [hunk 3]
```diff
@@ -1,21 +1,49 @@
-package org.bukkit.entity;
```
Comment (md-5) [inline src/main/java/org/bukkit/entity/Creeper.java:1]: Yeah formatting here too D:

Comment (stirante) [inline src/main/java/org/bukkit/entity/Creeper.java:1]: I have no idea how i've done it D: I'll fix it

File: src/main/java/org/bukkit/block/Beacon.java (hunk 2 above)
Comment (stirante) [inline src/main/java/org/bukkit/block/Beacon.java]: You are seriously? One space?

File: src/main/java/org/bukkit/inventory/ItemStack.java (hunk 1 above)
Comment (stirante) [inline src/main/java/org/bukkit/inventory/ItemStack.java:1]: I think it's GitHub's fault. When I was syncing there was some errors.

File: src/main/java/org/bukkit/block/Beacon.java (hunk 2 above)
Comment (ammaraskar) [inline src/main/java/org/bukkit/block/Beacon.java]: Yes, formatting is important.

**Comment by feildmaster** [comment]
Don't commit stuff that doesn't work. Thanks for suggesting api's though... I will add these myself, however.

Also, separate PR's for separate commits. Easier time viewing/merging changes

**Comment by stirante** [comment]
It was working before github broke code. I tested every method  and class.

**Pull request closed without merging**